spec/rails-settings-cached/setting_spec.rb

```diff
@@ -96,6 +105,11 @@
       expect(Setting.get_all).to include(:default1, :default2)
     end
 
+    it "should include namespace defaults" do
+      expect(RailsSettings::Default).to receive(:instance).and_return({ "test.default1" => 1, "test.default2" => '2', demo: 3 })
+      expect(Setting.get_all('test.')).to include(:'test.default1', :'test.default2')
```

Comment:
`Setting.get_all('test.')`

I don't want add this feature to let this gem more complex.